src/main/java/exam2/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,106 @@
+package exam2.lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoNumbers {
+
+    public final static int LOTTO_START_NUMBER = 1;
+    public final static int LOTTO_END_NUMBER = 45;
```

Comment:
아실수도 있겠지만 public static final 순서로 작성하는게 컨벤션입니다!
```suggestion
    public static final int LOTTO_START_NUMBER = 1;
    public static final int LOTTO_END_NUMBER = 45;
```